src/main/java/echoServer/http/Router.java

```diff
@@ -0,0 +1,16 @@
+package echoServer.HTTP;
+
+import echoServer.http.StatusCode;
```

Comment:
`StatusCodes` would work better I think, since this is a place that will presumably hold many codes.